.storybook/components/Wrapper/Wrapper.js

```diff
@@ -13,7 +13,7 @@ class Wrapper extends Component<Props> {
     const { children } = this.props;
     return (
       <OuterWrapper>
-        <InnerWrapper>{children}</InnerWrapper>
+        <InnerWrapper> {children} </InnerWrapper>{' '}
```

Comment:
looks like this was committed by mistake 🙃

Reply:
D'oh, knew I forgot something!